packages/donations-app/app/components/checkout/cardForm.html

```diff
@@ -65,7 +64,6 @@
         <input
           id="donate-expiry-date"
           type="text"
-          class="form-control"
           name="cardExpiryDate"
           placeholder="Expiration date (MM/YY)"
```

Comment:
I think we can implement automatic slash adding when the first 2 digits (month) of expiration date typed. It's not a comment to this PR, just a potential feature which I think is cool and user-friendly.

Reply:
Ok, pls suggest that to Teti.